blueprints/addon/files/ember-cli-build.js

```diff
@@ -0,0 +1,17 @@
+/* global require, module */
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
```

Comment:
i wonder if `ember-addon` and `ember-app` really want to be seperate modules... This hole NPM crap makes this so tricky